Pages/itemPage.js

```diff
@@ -47,83 +61,120 @@ class ItemPage{
     }
 
     validateUsersName(userName){
-        console.log("This method validates User's name after placed bid")
+        console.log("This method validates User's name after placed bid");
         return expect(userName).toBe(data.userAssert);
     }
 
     validateUsersBid(userBid){
-        console.log("This method validates User's bid price after placed bid")
+        console.log("This method validates User's bid price after placed bid");
         return expect(userBid).toBe(data.bidAssert);
     }
 
+    validateHigherBid(userBid){
+        console.log("This method validates User's bid price after placed bid");
+        return expect(userBid).toBe(data.bidAssertFiveThousandDollars);
+    }
+
+    validateBidWithDecimalPlaces(userBid){
+        console.log("This method validates User's bid price with decimal places after placed bid");
+        return expect(userBid).toBe(data.bidAssertWithDecimalPlaces);
+    }
+
     validateDateOfBid(bidDate){
         console.log("This method validates User's bid date after placed bid")
-        return expect(bidDate).toBe(`${data.d.getDate()} ${data.months[data.d.getMonth()]} ${data.d.getFullYear()}`)
+        return expect(bidDate).toBe(`${super.getDate().getDate()} ${data.months[super.getDate().getMonth()]} ${super.getDate().getFullYear()}`);
     }
 
     validateItemDetails(itemDetails){
-        console.log("This method validates Item details")
-        return expect(itemDetails).toBe(data.itemDetailsAssert)
+        console.log("This method validates Item details");
+        return expect(itemDetails).toBe(data.itemDetailsAssert);
     }
 
     waitForUserImg(){
-        console.log("This method waits for User's image to show up")
-        return browser.wait(EC.presenceOf($(this.userImgLocator)), 5000);
+        console.log("This method waits for User's image to show up");
+        return browser.wait(EC.presenceOf($(this.userImgLocator)), 7000);
     }
 
     getBidDate(){
-        console.log("This method gets date from User's placed bid")
+        console.log("This method gets date from User's placed bid");
         return this.bidDate.getText();
     }
 
     getBidPrice(){
-        console.log("This method gets price from User's placed bid")
+        console.log("This method gets price from User's placed bid");
         return this.bidPrice.getText();
     }
 
     getUserName(){
-        console.log("This method gets name from User's placed bid")
+        console.log("This method gets name from User's placed bid");
         return this.userName.getText();
     }
 
     getItemDetails(){
-        console.log("This method gets Item's details")
+        console.log("This method gets Item's details");
         return this.itemDetails.getText();
     }
 
     getBidConfirmationText(){
-        console.log("This method gets message that User placed a bid")
+        console.log("This method gets message that User placed a bid");
```

Comment:
Also, can you combine all methods which gets some kind of bid information into one method
"getBidInforamtion(informationType)" with switch case logic inside.
Try to have less methods with similar logic, and combine them into one wherever it is possible (this comment refers to all page object methods in this PR, please review them once again).

Reply:
I tried to reduce number of methods, and combine them in on methods(s)